### ISSUE TITLE
Lesson is not explained well

### ISSUE THREAD

**Issue by kylejohnson363** [learn-co-curriculum/dsc-2-23-09-resampling-methods-lab#4]
I am having a hard time following what this lab is asking to do, especially after looking at the solution for guidance.  The bootstrapping function does not even use the 'n' input called for in the problem and appears to only output a single response instead of n responses.  Then when applying the permutation in practice, the solution contains sample data that is not found in the lab or in the lab's repository.  It references the scenario from the previous lesson but the previous lesson does not appear to contain any data.  I grasp the concepts and follow what the solution is doing so I am moving on

**Comment by Redaisy** [comment]
Agreed. I don't think the bootstrapping works. The solution just returns a different sample of same length as your input sample, rather than taking random subsamples of the original sample. I think this works, but let me know otherwise:
def bootstrap(sample, n):
    lst_samples = []
    for i in range(0,n):
        sub_sample = np.random.choice(sample, size = int(len(sample)/n), replace = True)
        lst_samples.append(sub_sample)
    return lst_samples
